src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -108,12 +108,21 @@ public:
     // Use GetCommandId instead of commandId directly to avoid naming conflict with CommandIdentification in ExecutionOfACommand
     static constexpr CommandId GetCommandId() { return Commands::{{asUpperCamelCase name}}::Id; }
     static constexpr ClusterId GetClusterId() { return Clusters::{{asUpperCamelCase parent.name}}::Id; }
+    static constexpr bool kIsFabricScoped = {{isFabricScoped}};
 
     {{#zcl_command_arguments}}
     {{zapTypeToEncodableClusterObjectType type}} {{asLowerCamelCase label}}{{> cluster_objects_field_init}};
     {{/zcl_command_arguments}}
 
+    {{#if isFabricScoped~}}
+    {{#if (isServer source)}}
```

Comment:
What does it even mean for a server-source command to be fabric-scoped?  fabric-scoped in the spec for a command means "command must be associated with a fabric on the recipient of the command", no?   Is it actually used for any server-source commands, and what does it mean for those?

Reply:
🤷‍♂️ so in this PR, if you look at `TLSCertStruct`, it is marked as `isFabricScoped`, which is then used in `FindRootCertificateResponse`

I mean if an attribute can be fabric-scoped, I suppose it makes sense that a server-source command can too, right? (aren't they effectively the same, except attributes are pull & server commands are push?)

Here is that fabric scoped struct in spec: https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/f2136a2784837b93efb17814a57f2d4ef29d836a/src/tls/TLSCertificateManagement.adoc#ref_TLSCertStruct

I will defer to you here, I'm just trying to get the spec compiling 🤣